src/AutoUpdate/Settings.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace WPML\PB\AutoUpdate;
+
+class Settings {
```

Comment:
@strategio remember to add a PHPDoc here with a description explaining the purpose of this class:

- This is part of the "AutoUpdate" feature
- The "AutoUpdate" feature will be released in the next major version. We need a way for users to allow disabling it quickly, if necessary.

Reply:
Right, done!